README.md

```diff
@@ -1,2 +1,4 @@
-# guidelines
-Coding guidelines for the Le App project
+# Leapp project-wide guidelines
+
+[Python coding guidelines](python-coding-guidelines.md)
+[Contributing guidelines](contributing-guidelines.rst)
```

Comment:
Add newline between the links to make it clear there are 2 guidelines

Reply:
Added bullets instead.